### ISSUE TITLE
Travis broke

### ISSUE THREAD

**Issue by twiecki** [pymc-devs/pymc#1536]
```
======================================================================
ERROR: test_normal (pymc3.tests.test_examples.TestHierarchical)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/pymc-devs/pymc3/pymc3/tests/test_examples.py", line 242, in test_normal
    start = pm.find_MAP()
  File "/home/travis/build/pymc-devs/pymc3/pymc3/tuning/starting.py", line 77, in find_MAP
    dlogp = bij.mapf(model.fastdlogp(vars))
  File "/home/travis/build/pymc-devs/pymc3/pymc3/model.py", line 159, in fastdlogp
    return self.model.fastfn(gradient(self.logpt, vars))
  File "/home/travis/build/pymc-devs/pymc3/pymc3/memoize.py", line 15, in memoizer
    cache[key] = obj(*args, **kwargs)
  File "/home/travis/build/pymc-devs/pymc3/pymc3/theanof.py", line 60, in gradient
    return tt.concatenate([gradient1(f, v) for v in vars], axis=0)
  File "/home/travis/build/pymc-devs/pymc3/pymc3/theanof.py", line 60, in <listcomp>
    return tt.concatenate([gradient1(f, v) for v in vars], axis=0)
  File "/home/travis/build/pymc-devs/pymc3/pymc3/theanof.py", line 48, in gradient1
    return tt.flatten(tt.grad(f, v, disconnected_inputs='warn'))
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/gradient.py", line 553, in grad
    grad_dict, wrt, cost_name)
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/gradient.py", line 1317, in _populate_grad_dict
    rval = [access_grad_cache(elem) for elem in wrt]
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/gradient.py", line 1317, in <listcomp>
    rval = [access_grad_cache(elem) for elem in wrt]
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/gradient.py", line 1272, in access_grad_cache
    term = access_term_cache(node)[idx]
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/gradient.py", line 1106, in access_term_cache
    new_output_grads)
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/gof/op.py", line 700, in L_op
    return self.grad(inputs, output_grads)
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/tensor/elemwise.py", line 717, in grad
    sr = sr.dimshuffle(shuffle)
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/tensor/var.py", line 363, in dimshuffle
    pattern)
  File "/home/travis/minconda3/envs/testenv/lib/python3.5/site-packages/theano/tensor/elemwise.py", line 175, in __init__
    (input_broadcastable, new_order))
ValueError: ('You cannot drop a non-broadcastable dimension.', ((True, False), ('x', 'x', 0)))
-------------------- >> begin captured logging << --------------------
pymc3: DEBUG: Applied interval-transform to sg and added transformed sg_interval_ to model.
pymc3: DEBUG: Applied interval-transform to s and added transformed s_interval_ to model.
--------------------- >> end captured logging << ---------------------
```

**Comment by ColCarroll** [comment]
Working on figuring out the Theano pr that broke things:  tests pass installing at `dadb8af23e1fd453b21fe01b3167efbd270813d3`, but break at `380bcbb304aea63bbe7fa67942eed8cfe011fe23`.

**Comment by ColCarroll** [comment]
So it was https://github.com/Theano/Theano/commit/380bcbb304aea63bbe7fa67942eed8cfe011fe23

It references https://github.com/Theano/Theano/issues/790 

Checking out if there's a fix on our end...

**Comment by twiecki** [comment]
Good catch. Seems to only be that model so doesn't seem like a huge issue. Plus, seems like they implemented an optimization there which seems neat.

**Comment by ColCarroll** [comment]
Good point -- pushed[ a branch without that test](https://travis-ci.org/ColCarroll/pymc3/builds/177294651).  If I can't figure out how to fix the test, I'll open a PR here so we can at least unblock the build.

**Comment by springcoil** [comment]
I agree with Thomas on this. Not a huge issue, seems worth unblocking the build.

**Comment by ColCarroll** [comment]
Closed by #1537 .  Might be worth checking if it unbreaks in the future!
